api/src/main/resources/workbench-api.yaml

```diff
@@ -5091,6 +5122,14 @@ definitions:
       jwt:
         type: string
         description: the encoded/signed JWT
+  RasLinkRequestBody:
+    required:
+    - authcode
+    type: object
+    properties:
+      authcode:
+        type: string
+        description: the authorization code
```

Comment:
please describe this code.  What is its format, where does it come from, etc?

Reply:
Added some details. I am trying to say that is a standard Oauth code, that comes from RAS. Anything else worth to explain?

Reply:
OK